tensorrt_llm/rlhf_utils.py

```diff
@@ -0,0 +1,66 @@
+from typing import List
+
+import torch
+
+from tensorrt_llm._ray_utils import control_action_decorator
+from tensorrt_llm._torch.utils import get_device_uuid
+from tensorrt_llm._torch.virtual_memory import (materialize_with_tag,
+                                                release_with_tag,
+                                                verify_sleep_wakeup_tags)
+from tensorrt_llm.logger import logger
+
+
+class WorkerExtension:
+
+    @control_action_decorator
+    def sleep(self, sleep_tags: List[str]):
```

Comment:
If setting `enable_sleep=True` is a prerequisite for using sleep/wakeup(), can we add a check here or somewhere else and throw a ValueError if needed?